fiddler/manifest.json

```diff
@@ -0,0 +1,56 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "ee617671-508e-4bb3-ba25-8815b11a16aa",
+  "app_id": "fiddler",
+  "classifier_tags": [
+    "Supported OS::Linux",
+    "Supported OS::Mac OS",
+    "Supported OS::Windows",
+    "Category::Alerting",
+    "Category::Monitoring",
+    "Category::Metrics",
+    "Offering::Integration"
+  ],
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Fiddler integration to bring model metrics into DataDog",
+    "title": "Fiddler",
+    "media": [
+      {
+        "caption": "Fiddler Model Performance Management",
```

Comment:
We recommend adding images such as screenshots of dashboards or the Fiddler UI. We do not recommend adding the logo in the media carousel as it will appear elsewhere on your tile.

Reply:
Done.